simulation/src/xline_group.rs

```diff
@@ -127,3 +169,97 @@ impl Drop for XlineGroup {
         debug!("all nodes shutdowned");
     }
 }
+
+pub struct SimEtcdClient {
```

Comment:
Madsim has a `madsim-etcd-client` crate, could it be used here?

Reply:
`madsim-etcd-client` lacks many methods, such as watch